### PR TITLE
Counties class serialize data structure when being stored

### PR DESCRIPTION
**Pull request by JeffPyeBrook**
Workaround APC crash when retrieving transient.  A cleaned up version of a prior PR

### CODE REVIEW

File: wpsc-core/wpsc-includes.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-if ( defined( 'WPEC_LOAD_DEPRECATED' ) && WPEC_LOAD_DEPRECATED ) {
+if ( defined( 'WPEC_LOAD_DEPRECATED' ) ) {
```
Comment (JustinSainton) [inline wpsc-core/wpsc-includes.php]: What's the reason for this change?

File: wpsc-includes/misc.functions.php [hunk 2]
```diff
@@ -988,3 +988,76 @@ function _wpsc_remove_erroneous_files() {
...
+
+			// if there was a transient, but it could not be decoded, we delete the transient to get back
+			// to a working state
+			if ( false === $value ) {
+				_wpsc_delete_transient( $transient );
+			}
+		}
+	}
+
+	return $value;
```
Comment (JustinSainton) [inline wpsc-includes/misc.functions.php]: $value is not defined

File: wpsc-core/wpsc-includes.php (hunk 1 above)
Comment (JeffPyeBrook) [inline wpsc-core/wpsc-includes.php]: not sure why the cleanup rebase/cherry-pick/merge picked it up here ,   but WPEC_LOAD_DEPRECATED could be defined and be false

Comment (JustinSainton) [inline wpsc-core/wpsc-includes.php]: Yep, let's restore the `&& WPEC_LOAD_DEPRECATED`

File: wpsc-includes/misc.functions.php (hunk 2 above)
Comment (JeffPyeBrook) [inline wpsc-includes/misc.functions.php]: it is now

Comment (JeffPyeBrook) [inline wpsc-includes/misc.functions.php]: $value initialized
